src/controllers/main/main.ts

```diff
@@ -582,6 +583,7 @@ export class MainController extends EventEmitter {
         return this.isSignRequestStillActive
       }
     )
+    estimationErrorEmitter.init(this.signAccountOp)
```

Comment:
We synced up offline with Bobby and Simo, that introducing another approach for emitting sounds the easiest way to do this. In future, could be refactored if we came up with better approach.